controllers/resources.go

```diff
@@ -50,6 +51,14 @@ func getChartTemplates(r *SpecialResourceReconciler) (*unstructured.Unstructured
 
 func createImagePullerRoleBinding(r *SpecialResourceReconciler) error {
 
+	_, err := clients.Interface.RbacV1().ClusterRoles().Get(context.TODO(), "system:image-puller", metav1.GetOptions{})
```

Comment:
clients.Interface.Get did not work?

Reply:
Why is the handling not done in line 72?

Reply:
Good point